tinyformat_test.cpp

```diff
@@ -113,6 +114,20 @@ void speedTest(const std::string& which)
             fmt::print("{:.10f}:{:04}:{:+}:{}:{}:{}:%\n",
                 1.234, 42, 3.13, "str", (void*)1000, 'X');
     }
+    else if(which == "format::prepare")
+    {
+        // format::prepare version.
+        const auto prepared_format =
+            fmt::prepare<double, int, double, const char*, void*, char>("{:.10f}:{:04}:{:+}:{}:{}:{}:%\n");
+        fmt::basic_memory_buffer<char, 1024> buffer;
+        for(long i = 0; i < maxIter; ++i)
+        {
+            auto finished_at =
+                prepared_format.format_to(buffer, 1.234, 42, 3.13, "str", (void*)1000, 'X');
```

Comment:
Have you checked that this produces the same output as the `fmt::print` version?

Reply:
Yep, the outputs are the same:
```
format timings:
1.2340000000:0042:+3.13:str:0x3e8:X:%
real 0.00
user 0.00
sys 0.00
fmt::prepare timings:
1.2340000000:0042:+3.13:str:0x3e8:X:%
real 0.00
user 0.00
sys 0.00
```